File: src/boj_11720.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class boj_11720 {
    public static void main(String[] args) {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        try {
            int n = Integer.parseInt(br.readLine());
            String[] input = br.readLine().split("");
            int sum = 0;
            for (int i = 0; i < n; i++) {
                sum += Integer.parseInt(input[i]);
            }
            System.out.println(sum);
        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: src/boj_11720.kt
fun main() {
    val br = System.`in`.bufferedReader()

    val n = br.readLine().toInt()
    val numbers = br.readLine()
    var sum = 0
    for (i in 0..<n) {
        sum += numbers[i] - '0'
    }
    println(sum)
}